src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -59,8 +59,11 @@ public List<Dispute> getAllDisputes(
 			@RequestParam(required = false)
 			@DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
 			@Parameter(description = "If specified, will retrieve records older than this date (specified by yyyy-MM-dd)", example = "2022-03-15")
-			Date olderThan) {
-		Iterable<Dispute> allDisputes = disputeService.getAllDisputes(olderThan);
+			Date olderThan,
+			@RequestParam(required = false)
+			@Parameter(description = "If specified, will retrieve records which do not have the specified status", example = "CANCELLED")
+			DisputeStatus excludeStatus) {
```

Comment:
Interesting. Here in Java it's a param "DisputeStatus excludeStatus", but when openapi generates this it's "ExcludeStatus? excludeStatus"